tf2onnx/tfonnx.py

```diff
@@ -99,6 +99,12 @@ def tflist_to_onnx(node_list, shape_override):
                 continue
             elif a in ignored_attr:
                 continue
+            elif a in ["key_dtype", "value_dtype", "Tin", "Tout"]:
```

Comment:
is this needed ? Should this not be the same as the dtype if that specific input ?

Reply:
removed.